samples/face4.py

```
@@ -0,0 +1,92 @@
+import cv2
+import os
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)